core/modules/user/user.module

```diff
@@ -1413,8 +1420,7 @@ function user_menu() {
 
   $items['user/login'] = array(
     'title' => 'Log in',
-    'page callback' => 'backdrop_get_form',
-    'page arguments' => array('user_login'),
+    'page callback' => 'user_login_page',
```

Comment:
Is there any risk to this for contrib? Any chance a `_menu_alter()` exists expecting the current `page callback`? Not sure how significant this would be.

Reply:
It's possible, but generally I think low-risk. I like that this brings `user/login` inline with `user/password` and `user/register`, which both use dedicated page callbacks and not directly calling `backdrop_get_form()`.